test/ClangImporter/static_inline_serialize.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %empty-directory(%t)
+
+// Try serialization of clang-generated functions.
+
+// RUN: %target-swift-frontend -parse-as-library -module-name=static_inline -emit-module %S/Inputs/static_inline.swift -import-objc-header %S/Inputs/static_inline.h -o %t/static_inline.swiftmodule
+// RUN: %target-swift-frontend -module-name test -O -emit-ir %s -I %t | %FileCheck --check-prefix=CHECK-IR %s
+
+// CHECK: sil shared [serializable] [clang c_inline_func] @c_inline_func : $@convention(c) (Int32) -> Int32
```

Comment:
Did you mean to still have this CHECK line, when there's no FileCheck invocation for it?

Reply:
I added the FileCheck invocation and fixed the CHECK line, thanks